sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -233,7 +245,8 @@ export class Receiver {
    * @returns Promise<ServiceBusMessage | undefined>
    * - Returns `Message` identified by sequence number.
    * - Returns `undefined` if no such message is found.
-   * - Throws an error if the message has not been deferred.
+   * @throws Error if the underlying connection, client or receiver is closed.
+   * @throws MessagingError  if the service returns an error for the receiveDeferredMessage operation.
```

Comment:
```suggestion
   * @throws MessagingError if the service returns an error for the receiveDeferredMessage operation.
```

Reply:
updated

Reply:
This one got missed

Reply:
commit was missed on whitespaces and now updated, can check these again